tests/test_sessions.py

```diff
@@ -287,6 +287,14 @@ def test_run_install_only_should_install(self):
             **run_with_defaults(paths=mock.ANY, silent=True, env={}, external="error"),
         )
 
+    def test_run_dry_run(self):
+        session, _ = self.make_session_and_runner()
+
+        with mock.patch("nox.command.run", autospec=True) as run:
+            session.run(sys.executable, "--version", dry_run=True)
```

Comment:
This wasn't added. And I don't think it shovel be, though, I think is should be a command line flag only. Probably monkeypatch the value here instead.